package.json

```diff
@@ -51,7 +51,7 @@
     "@map-colonies/openapi-express-viewer": "^3.0.0",
     "@map-colonies/read-pkg": "0.0.1",
     "@map-colonies/schemas": "^1.0.1",
-    "@map-colonies/telemetry": "^5.4.0",
+    "@map-colonies/telemetry": "./map-colonies-telemetry-7.0.1.tgz",
```

Comment:
Change to proper version

Reply:
as the comment says: blocked by https://github.com/MapColonies/telemetry/pull/78